awx/ui_next/src/components/Lookup/CredentialLookup.jsx

```diff
@@ -35,6 +35,15 @@ function CredentialLookup({
   const [credentials, setCredentials] = useState([]);
   const [count, setCount] = useState(0);
   const [error, setError] = useState(null);
+  const isMounted = useRef(null);
+
+  useEffect(() => {
+    isMounted.current = true;
+    return () => {
+      isMounted.current = false;
+    };
+  }, []);
```

Comment:
Can we convert this to useRequest to avoid all the isMounted logic?

Reply:
Yes, I missed that one! I converted ProjectLookup and InventoryLookup but the CredentialLookup slipped through.